front/src/common/redux/groups/groups.saga.js

```diff
@@ -0,0 +1,57 @@
+import { put, call, all, takeEvery } from 'redux-saga/effects';
+import { updateGroup, loadGroupsData, deleteStudent, getOneGroup } from './groups.api';
+import * as types from './types';
+
+export function* updateSaga(payload) {
+    try {
+        const response = yield call(updateGroup, payload);
+        yield put({type: types.SET_UPDATED_CURRENT_GROUP, data: response.data});
```

Comment:
Please, use `payload` key instead of `data` and do it for all actions create as a part of this merge request

Reply:
done, fix in groups actions